codice/ddf
# DDF-5786 Adds beta admin UI to the web context policy manager

#### What does this PR do?
- Adds beta admin UI to the web context policy manager
- Removes IDP Server configuration from the installer

#### Who is reviewing it? 
@bakejeyner @SmithJosh 

#### Select relevant component teams: 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@garrettfreibott
@stustison 

#### How should this be tested?
- Verify the admin UI and the beta admin UI installer works
- Verify logging in to the admin console through the beta admin UI works

#### What are the relevant tickets?
Fixes: #5786 

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
